fhireval/test_suite/bulk_export/test_bulk_export.py

```diff
@@ -1,13 +1,118 @@
 import pytest
 import pdb
+from fhireval.data import prep_server, id_list
+from collections import defaultdict
+from time import sleep
+from pprint import pformat
+import json
+from base64 import b64decode
 
-test_id = f"{'2.6.1':<10} - Bulk Export "
+test_id = f"{'2.9.1':<10} - Bulk Export "
 
 test_weight = 2
 
+def bulk_export(host, endpt = None, resources=None):
+    # Request our data to be fhir+json separated by new lines
+    if endpt is None or endpt == "":
+        endpoint = "$export?_outputFormat=application%2Ffhir%2Bndjson"
```

Comment:
Do you have to pre-escape this string?

Reply:
The plus must have another meaning, because it does require it. I've updated them to use the forward slash, which does still work.